sklearn/decomposition/tests/test_nmf.py

```diff
@@ -254,25 +385,28 @@ def test_nmf_sparse_transform():
 def test_non_negative_factorization_consistency(init, solver, alpha_W, alpha_H):
     # Test that the function is called in the same way, either directly
     # or through the NMF class
+    max_iter = 500
     rng = np.random.mtrand.RandomState(42)
     A = np.abs(rng.randn(10, 10))
     A[:, 2 * np.arange(5)] = 0
 
-    W_nmf, H, _ = non_negative_factorization(
+    W_nmf, H, n_iter = non_negative_factorization(
```

Comment:
Was this change and the one bellow introduced to check equality between the two `(H, n_iter)`-pairs? This assertion is not present (yet?).

Reply:
unwanted change. Removed